Linux/src/main.cpp

```diff
@@ -5,13 +5,6 @@
 // Copyright   : Your copyright notice
 // Description :
 //============================================================================
-
-
-//macro to write shader programs inline
-#ifndef GLSL 
-#define GLSL(version,A) "#version " #version "\n" #A
-#endif
-
```

Comment:
This is still defined somewhere else - in an old file no doubt. Move the definition to game utils, so the game author can use if intending to use GLSL compile.